lib/cadet/assessments/assessments.ex

```diff
@@ -727,6 +775,14 @@ defmodule Cadet.Assessments do
     |> Repo.update()
   end
 
+  def update_submission_status_router(submission = %Submission{}, question = %Question{}) do
+    if question.type == :voting do
+      update_contest_voting_submission_status(submission, question)
+    else
+      update_submission_status(submission, question.assessment)
+    end
+  end
+
```

Comment:
Please use a guard instead of this.

(Anyway, `update_contest_voting_submission_status` will change if the assumption/limitation that a voting assessment can only have one question is removed.)